src/App.tsx

```diff
@@ -1,11 +1,55 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import Footer from './components/Footer';
+import TodoList from './components/TodoList';
 
-const USER_ID = 0;
+export const userId = 10590;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filterType, setFilterType] = useState('all');
+
+  const fetchTodos = () => {
+    client
+      .get<Todo[]>(`/todos?userId=${userId}`)
+      .then((receivedTodos) => {
+        setTodos(receivedTodos);
+      })
+      .catch(() => {
+        setError('Unable to fetch todos data');
+      });
+  };
+
+  useEffect(() => {
+    fetchTodos();
+  }, []);
+
+  const handleFilterChange = (type: string) => {
+    setFilterType(type);
+  };
+
+  const handleCloseError = () => {
+    setError(null);
+  };
+
+  useEffect(() => {
+    if (error) {
+      const timer = setTimeout(() => {
+        setError(null);
+      }, 3000);
+
+      return () => {
+        clearTimeout(timer);
+      };
+    }
+
+    return () => {};
```

Comment:
do you need this return here? 🤔

Reply:
Expected to return a value at the end of arrow function.